system-addon/test/functional/mochitest/head.js

```diff
@@ -13,6 +16,11 @@ const ACTIVITY_STREAM_PREF = "browser.newtabpage.activity-stream.enabled";
 pushPrefs([ACTIVITY_STREAM_PREF, true]);
 gBrowser.removePreloadedBrowser();
 
+async function clearHistoryAndBookmarks() { // eslint-disable-line no-unused-vars
+  await PlacesTestUtils.clearHistory();
```

Comment:
Apparently this is deprecated:
> > +  await PlacesTestUtils.clearHistory();
> nit: Please await PlacesUtils.history.clear(); instead, the PlacesTestUtils shortcut is deprecated.

https://bugzilla.mozilla.org/show_bug.cgi?id=1410240#c17